.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,34 @@
+{
+    // See https://aka.ms/vscode-remote/devcontainer.json for format details
```

Comment:
JSON can't have comments (unless some parser supports it, but then it's nonstandard)

Reply:
This from the reference file at
https://github.com/microsoft/vscode-dev-containers/blob/master/container-templates/docker-compose/.devcontainer/devcontainer.json

Reply:
I can't see anyone else needing to read this file, so if this is what VSCode does then we can keep it that way.